__tests__/redux/actions/authActions.test.js

```diff
@@ -89,4 +90,16 @@ describe('Should make it happen', () => {
       expect(store.getActions().length).toEqual(3);
     });
   });
+  test('should be able to log out', () => {
+    moxios.wait(() => {
+      const request = moxios.requests.mostRecent();
+      request.respondWith({
+        status: 200,
+        message: ' you are now logged out',
+      });
+    });
+    return store.dispatch(loggOut('loggOut')).then(() => {
```

Comment:
Does your action have a parameter? Why are you dispatching `loggOut('loggOut')` ?

Reply:
okay, corrected it